using/src/request.rs

```diff
@@ -0,0 +1,62 @@
+//! A `Request` for a spec within a document.
+//!
+//! `Request`s are derived from `Directive`s during schema bootstrapping.
+use graphql_parser::{
+    schema::{Directive, Value},
+    Pos,
+};
+
+use crate::spec::{Spec, SpecParseError};
+
+/// Requests contain a `spec`, the `prefix` requested for that spec (which
```

Comment:
nit: for doc comments, it's usually really nice to have the first line to be a bit shorter and then an added break. When the docs get built the first line is used as a description for that particular exported type. This is what it looks like now for `Request` vs `Pos`
<img width="967" alt="Screen Shot 2021-01-19 at 19 12 34" src="https://user-images.githubusercontent.com/8107784/105075769-82b7c380-5a8a-11eb-9c11-a0022e79b2c4.png">